shell/platform/common/cpp/client_wrapper/include/flutter/texture_registrar.h

```diff
@@ -42,10 +42,47 @@ class PixelBufferTexture {
   const CopyBufferCallback copy_buffer_callback_;
 };
 
+// A gpu buffer texture.
```

Comment:
Will you suggest this part to flutter?
Do you have any plans on how to do that? (just curiosity :smile: )